SpatialGDK/Source/SpatialGDK/Public/Interop/ActorSystem.h

```diff
@@ -38,14 +38,22 @@ struct ActorData
 
 struct FObjectRepNotifies
 {
-	FObjectRepNotifies(const TWeakObjectPtr<UObject> WeakObjectPtr)
-		: Object(WeakObjectPtr)
-	{
-	}
-	TWeakObjectPtr<UObject> Object;
+	UObject* Object;
```

Comment:
Are we sure we want to use a raw UObject pointer here? Are these pointers guaranteed to be store and thrown away in a single frame?

Reply:
they are but I guess the object could be killed by a repnotify (though presumably gc wouldn't have run as this is synchronous...), but better safe than sorry - changed to use a tweakobjectptr